suite-native/blockchain/src/blockchainThunks.ts

```diff
@@ -0,0 +1,103 @@
+import { createThunk } from '@suite-common/redux-utils';
+import { NetworkSymbol, isNetworkSymbol } from '@suite-common/wallet-config';
+import {
+    blockchainActions,
+    fetchAndUpdateAccountThunk,
+    selectDeviceAccountByDescriptorAndNetworkSymbol,
+    selectAccountsByNetworkSymbol,
+    selectAccountsSymbols,
+    subscribeBlockchainThunk,
+} from '@suite-common/wallet-core';
+import { AccountKey } from '@suite-common/wallet-types';
+import { getNetwork } from '@suite-common/wallet-utils';
+import { BlockchainNotification } from '@trezor/connect';
+
+const actionsPrefix = '@native/blockchain';
+
+const accountLastFetchTime: Record<AccountKey, number> = {};
+const ACCOUNT_LAST_FETCH_TIME_LIMIT_MS = 1000 * 10;
```

Comment:
```suggestion
const ACCOUNT_LAST_FETCH_TIME_LIMIT_MS = 10_000;
```

I believe this is more readable, but up to you 😅